src/firebase-config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "apiKey": "AIzaSyDk7VMMBwr6bqDsrzbUYGpnthHaMioYe2s",
+  "authDomain": "chat-app-15946.firebaseapp.com",
+  "projectId": "chat-app-15946",
+  "storageBucket": "chat-app-15946.appspot.com",
+  "messagingSenderId": "533178884935",
+  "appId": "1:533178884935:web:ef9483d776d0107ffe6262",
+  "measurementId": "G-P58VBCQ0SY"
+}
```

Comment:
not able to send message by this config detail but my config detail works fine when I changed it. Can't accept merge request.

Reply:
Could you try to fix this?